pkg/apis/metering/v1/report.go

```diff
@@ -34,6 +34,9 @@ type ReportSpec struct {
 	// Schedule configures when the report runs.
 	Schedule *ReportSchedule `json:"schedule,omitempty"`
 
+	// Expiration of the report. Report will be deleted once this date has passed
+	Expiration *meta.Duration `json:"expiration,omitempty"`
+
```

Comment:
Anytime we add any fields to any of the CRD APIs, we need to mirror those changes to the corresponding CRD manifest (`charts/metering-ansible-operator/templates/crds/...`) and run `make metering-manifests`.

Reply:
done in 
0c8e8d6